.gitignore

```diff
@@ -4,3 +4,4 @@ designkit
 node_modules
 package-lock.json
 yarn.lock
+yarn-error.lock
```

Comment:
This is PR contains a really nice feature, looking forward to use it! But I need to be a bit picky, how is the 'yarn-error.lock' file connected to this feature?
And when did we switch to yarn? :D

Reply:
@Jumace We didn't officially. Currently, some use npm, some yarn. I even use both :-D

Reply:
done.